pipeline/Makefile

```diff
@@ -145,10 +142,15 @@ BG = java -server -XX:+UseG1GC -Xmx$(BGMEM) -cp $(BGJAR) com.bigdata.rdf.store.D
 # Load blazegraph
 
 BG_PORT = 8899
```

Comment:
That's true. For the moment though this isn't being actively used. Once we have sparta checking integrated though, we should definitely think about that.